eng/Versions.props

```diff
@@ -28,7 +28,14 @@
     <UsingToolXliff>false</UsingToolXliff>
     <UsingToolNetFrameworkReferenceAssemblies>true</UsingToolNetFrameworkReferenceAssemblies>
     <!-- Build tools -->
-    <MicrosoftNetCompilersVersion>3.0.0</MicrosoftNetCompilersVersion>
```

Comment:
So we are using whatever the arcade/the SDK default compiler version is? If it is greater than 3.0.0, that's ok.

Reply:
Yeah, it's 4+